app/views/layouts/blacklight/base.html.erb

```diff
@@ -41,6 +41,7 @@
     <% end %>
 
     <link rel="stylesheet" href="/sul-components/v2024-08-01/styles/sul.css" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
```

Comment:
could we do these as `@import url()`s in the main stylesheet, so that everything is in one place?

Reply:
Done.